### DISCUSSION TITLE
data get ordered without specific query

### DISCUSSION THREAD

**Discussion by rifqi-ridwan** [go-rel/rel#312] [Q&A] [answered]
is it right that `rel.FindAll()` ordered data by `updated_at` field?
since i'm not doing any specific sort, but my data always return sorted by `updated_at` instead `id`
but the instrumentation didn't show if it's sorted by `updated_at`. i just couldn't find out where the sort comes from.

**Comment by Fs02** [comment, marked as answer]
Can you try issuing the same query directly to db and compare the result?

**Reply by rifqi-ridwan** [reply]
my bad sorry
